File: app/src/main/java/com/moudle/app/ClipDrawableActivity.java
package com.moudle.app;

import android.graphics.drawable.ClipDrawable;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.ImageView;

/**
 * ClipDrawable         clip  对图片进行裁剪操作
 */
public class ClipDrawableActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_clip);

        ImageView iv_clip = findViewById(R.id.iv_clip);

        ClipDrawable clipDrawable = (ClipDrawable) iv_clip.getDrawable();
        clipDrawable.setLevel(8000);
    }
}

File: app/src/main/java/com/moudle/app/activity/ClipDrawableActivity.kt
package com.moudle.app.activity

import android.graphics.drawable.ClipDrawable
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.moudle.app.databinding.ActivityClipBinding

/**
 * ClipDrawable         clip  对图片进行裁剪操作
 */
class ClipDrawableActivity : AppCompatActivity() {
    private lateinit var clipBinding: ActivityClipBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        clipBinding = ActivityClipBinding.inflate(layoutInflater)
        setContentView(clipBinding.root)

        val clipDrawable = clipBinding.ivClip.drawable as ClipDrawable
        clipDrawable.level = 8000
    }
}